examples/bool.kl

```diff
@@ -0,0 +1,64 @@
+#lang "n-ary-app.kl"
+
+(import (shift "n-ary-app.kl" 1))
+(import (shift "quasiquote.kl" 1))
+
+(define not
+  (lambda (b)
+    (if b #false #true)))
+
+(define binary-and
+  (lambda (x y)
+    (if x y #false)))
+
+(define binary-or
```

Comment:
I initially wanted to name those `&&` and `||`, but it looks like `|` is not allowed in an identifier? Is that special syntax for something?

Reply:
Right now, Klister uses R6RS Scheme rules for identifiers, because it seemed to be a reasonable default. In Scheme, `|` is a delimiter for identifiers that contain non-identifier characters, like back-tick in F#. So I could name a function `|My awesome function!!!|`. I think we can just turn `|` into an identifier char with no prob if you want.

Reply:
I'm fine either way, thanks for the explanation.